projects/Word_Guessing_Game/word_guessing_game.py

```diff
@@ -0,0 +1,54 @@
+import random
+
+with open("D:\Projects\PythonProjects\python_mini_projects\hangman_puzzle\words.txt", 'r') as f:
```

Comment:
provide relative path of words.txt otherwise exception will generate for file not found